agent/cmd/determined-agent/init.go

```diff
@@ -151,7 +151,4 @@ func registerAgentConfig() {
 		"Max attempts agent has to reconnect")
 	registerInt(flags, name("agent-reconnect-backoff"), defaults.AgentReconnectBackoff,
 		"Time between agent reconnect attempts")
-
-	registerString(flags, name("container-runtime"), defaults.ContainerRuntime,
```

Comment:
I think we should leave this flag in but only allow docker as the ```container-runtime```

If for some reason someone has ```determined-agent --container-runtime=docker``` I don't think we should break that use case